recipes/mold/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.45.0"
+
+class MoldConan(ConanFile):
+    name = "mold"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rui314/mold/"
+    license = "Zlib"
+    description = ("mold is a faster drop-in replacement for existing Unix linkers. It is several times faster than the LLVM lld linker")
+    topics = ("mold", "ld", "linkage", "compilation")
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    generators = "make"
+
+    def validate(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration('{} can\'t be built on {}.'.format(self.name, self.settings.os))
```

Comment:
```suggestion
            raise ConanInvalidConfiguration("mold can't be built on Windows")
```

Reply:
My line is generic and "copy-paste safe".

Reply:
YAGNI. Windows support is planned anyway.

Reply:
I need copy-pasting code snippets and do it all the time ;)